moped-editor/src/components/GridTable/SearchBar.js

```diff
@@ -160,6 +160,7 @@ const SearchBar = ({
     <>
       <TextField
         fullWidth
+        autoFocus
```

Comment:
i'm curious if y'all like this change! i will ask the MUG as well.

in an ideal world, the project list scroll position would persist when you navigate back to it. this `autoFocus` would break that, _I think_.

in order to preserve scroll position we would need to start using the new [data routers](https://reactrouter.com/en/main/routers/picking-a-router). i should probably open an issue for that....